common/app/views/support/Commercial.scala

```diff
@@ -89,6 +89,22 @@ object Commercial {
     allSponsors map (_ map escapeJavaScript)
   }
 
+  def getSponsorForGA(page: Page, key: String)(implicit request: RequestHeader): Html = Html {
+    if (isBrandedContent(page)) {
+      listSponsorLogosOnPage(page) match {
+        case Some(logos) => s"&$key=${logos.mkString("|")}"
+        case _ => ""
+      }
+    } else { "" }
+  }
+
+  def getBrandingTypeForGA(page: Page, key: String)(implicit request: RequestHeader): Html = Html {
```

Comment:
Does the key ever change?

Reply:
Hopefully not...?

I had a bit of an internal debate here, but was ultimately tempted into passing the keys/fields as parameters from the template, because I wanted them to be visible to the next person making changes to the querystring.

Reasoning why I would rather hardcode the keys in the GA file than `Commercial.scala`:
- It may not be immediately apparent to anyone altering the querystring that `cd31` and `cd42` are potentially present; passing them as parameters ensures they appear in `google.scala.html`
- Keeps the weirdness contained to `google.scala.html`. A html template using various Scala methods to generate a string that happens to parse as JSON? 😱 It already seems to be responsible for holding a lot of the GA logic, so I didn't want to spread it around.
- If anyone wants to change the key, they only have to modify `google.scala.html`

All loosely held convictions; happy to change it around 😁

Reply:
👍